ci/deploy-to-uat

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Echo all commands just before they are executed
+set -x
+
+# Stop processing this script if any of the commands we run return an error
+# (i.e. exit with a non-zero status)
+set -e
+
+cd /var/www
+
+# This script attempts a git fetch in CWP, waits for it to be ready
+# and creates a deployment request.
+
+# Creates a git fetch request
+curl -X POST -u $CWP_DASH_USER:$CWP_DASH_TOKEN \
+  https://dash.cwp.govt.nz/naut/project/$CWP_STACK_ID/git/fetches \
+  -H 'Content-Type: application/json' -H 'Accept: application/json' >response.json
+
+# We extract the request's status URL from the response
+LINK=$(cat response.json | jq -r '.data.links.self')
```

Comment:
🐘 typically I use lowercase for variables that are not exported/global, to make it more obvious script-local vs env variables.